bleachbit/Windows.py

```diff
@@ -338,6 +338,16 @@ def elevate_privileges(uac):
     return False
 
 
+def _add_command_line_parameters(parameters):
+    """
+    Add any command line parameters such as --debug-log.
+    """
+    if '--context-menu' in sys.argv:
+        return '{} {} "{}"'.format(parameters, ' '.join(sys.argv[1:-1]), sys.argv[-1])
```

Comment:
@rados 
Would you please write a brief note here as a comment to explain why it is necessary to format it another way?

Reply:
Sure, because the file path looses its quotes and if it contains spaces it becomes a problem.